workshop/porter-tf-aci/azure/bundle/porter.yaml

```diff
@@ -48,6 +48,17 @@ parameters:
   - name: database-name
     type: string
 
+outputs:
+  - name: IP_ADDRESS
+    type: string
+
+imageMap:
```

Comment:
omg we are using image map? just for fun or?

Reply:
ITS FOR REALZ! I mean for completeness.....

I was thinking we should add the ability to wire up images though! like in this ACI thing, it would be cool if i could jsut say ${bundle.images.WHATEVER} and we could figure out how to deal with that crazy relocation stuff later if needed

Reply:
I demand an issue to 🚲 🏠